plugins/org.eclipse.n4js.flowgraphs/src/org/eclipse/n4js/flowgraphs/model/ComplexNode.java

```diff
@@ -170,6 +178,11 @@ public Node getExit() {
 		return exit;
 	}
 
+	/** @return a node with a {@link JumpToken} or null */
+	public Node getJump() {
```

Comment:
Why JumpToken? I do not see any thing related token in the signature.

Reply:
It is returning a Node that has a JumpToken element